filecoin-proofs/src/api.rs

```diff
@@ -263,6 +373,51 @@ pub fn verify_seal(
     .map_err(Into::into)
 }
 
+/// Verify the provided piece inclusion proof demonstrates the piece commitment exists in a
+/// merkle tree of a specific size with root hash comm_d
```

Comment:
I think some words or punctuation are missing from this sentence. I know what this function does, but I don't quite understand the description here.

Reply:
@sidke - How about:

```
/// Verify that the provided PIP demonstrates that the piece whose commitment is `comm_p` is included in the data whose commitment is `comm_d`
```